odoo/modules/registry.py

```diff
@@ -244,7 +247,10 @@ def setup_models(self, cr):
             for model in env.values():
                 model._unregister_hook()
 
+        # clear cache to ensure consistency, but do not signal it
         self.clear_caches()
+        self.cache_invalidated = False
+
```

Comment:
This one is just noisy, since `registry_invalidated` is set to `True` (on purpose), and signaling will clear both flags anyway.